### DISCUSSION TITLE
Disabled days can be selected via keyboard navigation

### DISCUSSION THREAD

**Discussion by McSam27** [gpbl/react-day-picker#1477] [Support] [answered]
Hi there,

I'm not sure if this is intended... but when using the keyboard to navigate the calendar UI, you can press "Enter" to select a disabled day.

We are using the latest release of V8. (8.0.7 at the time of this thread)

Simple Codesandbox for this: 
https://codesandbox.io/s/sandpack-project-forked-5nmu1q?file=/App.tsx:521-522
Steps:
1. Navigate to the days with the keyboard
2. Press enter on a non-disabled day (the day becomes selected)
3. Use arrow keys to navigate to disabled day. Press "enter". Now this is selected.

I've seen some other people discussing if it's possible to "skip over" a day when the day is disabled, and navigating with the keyboard. I don't know if that's the best idea since that may confuse users, since they might be expecting to focus on the disabled day. Causing the user's focus to jump around.

I think this occurred in v7. And, we wrote a custom `if` statement to return from the `onDayClick` function if the day clicked/keypressed had the `modifiers.disabled` to mitigate this.

I'm wondering if this could turn into an issue, or if we are expected to handle keypresses on disabled days on our own?

Thanks! 

P.S. Loving the V8 implementation so far!

**Comment by gpbl** [comment, marked as answer]
@McSam27 thanks for the heads up! This should be fixed with v8.1.2 - give a look!

**Reply by McSam27** [reply]
You're the best!!! @gpbl
